classes/services/IssueService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file classes/services/IssueService.php
+*
+* Copyright (c) 2014-2017 Simon Fraser University
+* Copyright (c) 2000-2017 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* @class IssueService
+* @ingroup services
+*
+* @brief Helper class that encapsulates issue business logic
+*/
+
+namespace App\Services;
+
+class IssueService {
+
+	/**
+	 * Determine if a user can access galleys for a specific issue 
+	 * 
+	 * @param \Journal $journal
+	 * @param \Issue $issue
+	 * 
+	 * @return boolean
+	 */
+	public function userHasAccessToGalleys(\Journal $journal, \Issue $issue) {
```

Comment:
This is just a helper function to avoid copy/paste in case I need that logic again. The method still depends on IssueAction. 

Just to make sure I understand correctly, are you suggesting to move the logic directly into `api/v1/issues/IssuesHandler::getIssue()` directly instead of calling this helper method?